src/main/scala/no/ndla/audioapi/controller/AudioController.scala

```diff
@@ -407,5 +408,40 @@ trait AudioController {
       }
 
     }
+
+    get(
+      "/tag-search/",
+      operation(
+        apiOperation[TagsSearchResult]("getTags-paginated")
+          .summary("Retrieves a list of all previously used tags in articles")
+          .description("Retrieves a list of all previously used tags in articles")
+          .parameters(
+            asHeaderParam(correlationId),
+            asQueryParam(query),
+            asQueryParam(pageSize),
+            asQueryParam(pageNo),
+            asQueryParam(language)
+          )
+          .responseMessages(response403, response500)
+          .authorizations("oauth2"))
+    ) {
+      authUser.assertHasId()
+      authRole.assertHasRole(RoleWithWriteAccess)
+      val query = paramOrDefault(this.query.paramName, "")
```

Comment:
Gir det meining å berre tillate tag-søk for innlogga? Vanlig søk er vel åpent for alle så kvifor ikkje tags?

Reply:
Stjal kode for endepunktet fra draft-api så skrev om dette uten å tenke meg om. Men seff kan vi søke uten innlogging her!
Will fix :ok_hand: